### ISSUE TITLE
Pre-work for Grooming

### ISSUE THREAD

**Issue by mmiddaugh** [department-of-veterans-affairs/va.gov-team#13778]
## Issue Description
Some issues need time to review before grooming but do not need a complete discovery effort. Due to lack of familiarity, this is particularly true for VAMC template issues and some 508/accessibility changes.

The following issues need to be reviewed so that Facilities team engineers have a basic understanding of the code and feel prepared to groom on 9/30:

**Front end engineering**
-  ~~[#5473](https://github.com/department-of-veterans-affairs/va.gov-team/issues/5473) The VAMC /about page has three download PDF links at the end of the page which all have aria-label attributes that just read Download. Screen reader users are left to figure out "Download what?" because the accessible (aria) label is the only thing read.~~ (_now in validate_)
- ~~#[8964](https://github.com/department-of-veterans-affairs/va.gov-team/issues/8964) For the current page CONSIDER adding 'aria-current="page"` attribute to the sidenav link so screen reader users understand which page they are currently visiting~~ _(closed by Jennifer this week!)_
- [#6009](https://github.com/department-of-veterans-affairs/va.gov-team/issues/6009) Make it possible to view tables (on VAMC pages) at 400% zoom levels
- [#12970](https://github.com/department-of-veterans-affairs/va.gov-team/issues/12970) Dynamic query process for building Left Nav
- [#13026](https://github.com/department-of-veterans-affairs/va.gov-team/issues/13026) VBA names

**Back end engineering**
- [#13026](https://github.com/department-of-veterans-affairs/va.gov-team/issues/13026) VBA names

---
## Tasks
- [x] **Quick** code review/familiarize yourself with the issue described  
- [x] Identify dependencies
- [x] Identify questions for other teams

## Acceptance Criteria
- [x] Engineers are prepared to groom the issue
- [x] Unknowns are identified
- [x] Dependenices are identified

---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `tools-be`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by mpelzsherman** [comment]
Pushed up a PR for #5473: https://github.com/department-of-veterans-affairs/vets-website/pull/14377

**Comment by mpelzsherman** [comment]
Both #8964 and #6009 appear to be fixed (or I don't know how to reproduce).
